app/Http/Controllers/CompetitionsController.php

```diff
@@ -137,4 +137,17 @@ public function message(Competition $competition, Contest $contest)
 
         return view('messages.show', compact('messages', 'competition'));
     }
+
+    /**
+     * Display the specified leaderboard.
+     *
+     * @param  \Gladiator\Models\Competition  $competition
+     * @return \Illuminate\Http\Response
+     */
+    public function leaderboard(Competition $competition)
+    {
+        $leaderboard = $this->manager->getLeaderboard($competition, $competition->users);
```

Comment:
Gotcha. Just meant to grab them in controller and pass them along as one built out larger object w/ all the info inside of it.